server/src/main/java/io/crate/replication/logical/action/ReplayChangesAction.java

```diff
@@ -163,10 +162,7 @@ void performOnPrimary(IndexShard primary,
                     failure.accept(e);
                     return;
                 }
-                if (engineResult.getResultType() == Engine.Result.Type.MAPPING_UPDATE_REQUIRED ||
-                    (engineResult.getResultType() == Engine.Result.Type.FAILURE &&
-                     engineResult.getFailure() instanceof StrictDynamicMappingException)
-                ) {
+                if (engineResult.getResultType() == Engine.Result.Type.MAPPING_UPDATE_REQUIRED) {
```

Comment:
Just to verify I understand this right: This is because `StrictDynamicMappingException` was never used and is removed, making the old condition here dead code?

Reply:
Yes, exactly.